Fix variable declarations/refs in JS Set object doc

CreaTorAlexander opened the pull request:
Fixes #2263 I wasn't sure about the console.log()'s and the naming. I named it now mySet1 and mySet2, I thought it's not good to name it mySet and mySet2

CreaTorAlexander committed:
Fixes #2263

avivmu commented on files/en-us/web/javascript/reference/global_objects/set/index.html:
```diff
@@ -81,32 +81,32 @@ <h2 id="Examples">Examples</h2>
 <h3 id="Using_the_Set_object">Using the Set object</h3>
 
-<pre class="brush: js notranslate">let mySet = new Set()
+<pre class="brush: js notranslate">let mySet1 = new Set()
```
Also can use `const` since reference doesn't change, same with `mySet2`

CreaTorAlexander replied:
Agree. But then we should also update setA, setB, setC, intersection, difference and the array to const.
How can I update now an existing pull request?

Ryuno-Ki replied:
>  How can I update now an existing pull request?

Commit on the same branch and push your changes. GitHub will deal with the rest.

avivmu commented:
Push to the same branch

On Thu, Feb 11, 2021, 09:33 Alexander Heß <notifications@github.com> wrote:

> *@CreaTorAlexander* commented on this pull request.
> ------------------------------
>
> In files/en-us/web/javascript/reference/global_objects/set/index.html
> <https://github.com/mdn/content/pull/2264#discussion_r574293242>:
>
> > @@ -81,32 +81,32 @@ <h2 id="Examples">Examples</h2>
>
>  <h3 id="Using_the_Set_object">Using the Set object</h3>
>
> -<pre class="brush: js notranslate">let mySet = new Set()
> +<pre class="brush: js notranslate">let mySet1 = new Set()
>
> Agree. But then we should also update setA, setB, setC, intersection,
> difference and the array to const.
> How can I update now an existing pull request?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/mdn/content/pull/2264#discussion_r574293242>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEXDBNNZQDXH5YUXQZES2U3S6OB4TANCNFSM4XNPROWQ>
> .
>

CreaTorAlexander committed:
Fixes #2263

Updated the let's to const's because they never get reassigned.
Remove the console log's because at mySet1 there weren't any console.logs at the same example'
```
mySet1.has(o)       // true
mySet1.size         // 5
``` 
But it is a good question when do we use log's in the examples and when not.
